packages/esm-patient-immunizations-app/src/immunizations/immunizations-form.component.tsx

```diff
@@ -42,15 +53,35 @@ const ImmunizationsForm: React.FC<DefaultWorkspaceProps> = ({ patientUuid, close
   const [formState, setFormState] = useState(initialState);
   const updateSingle = <T extends keyof ImmunizationFormState>(name: T, value: typeof formState[T]) =>
     setFormState((state) => ({ ...state, [name]: value }));
+  const onChangeComboBox = (event) => {
```

Comment:
A couple of things here:

1. Wrap this function in [`useCallback`](https://reactjs.org/docs/hooks-reference.html#usecallback), as it's being passed to a stateful component.
2. When using this function, it can just be written: `onChange={onChangeComboBox}`
3. I notice in most places, we just directly call `updateSingle()` in the event hander. Why do we need this special handling here? (If there's no selected item, surely `vaccineUuid` should be set to, e.g., `null`)